frontend/packages/console-shared/src/components/status/status.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import {
+  HourglassStartIcon,
+  HourglassHalfIcon,
+  SyncIcon,
+  BanIcon,
+  WarningTriangleIcon,
+  UnknownIcon,
+} from '@patternfly/react-icons';
+import {
+  StatusIconAndText,
+  ProgressStatus,
+  ErrorStatus,
+  SuccessStatus,
+  InfoStatus,
+} from './status-components';
+
+export type StatusProps = {
+  status?: string;
+};
+
+export const Status: React.FC<StatusProps> = ({ status, children }) => {
+  switch (status) {
```

Comment:
It seems that the concepts of `status` and `title` are merged together at the moment, carried over from the existing `StatusIconAndText` component design.

With this `Status` component, we could map states (statuses?) to titles if we ever decide to decouple these concepts. In general, the `status` string should be the key indicator to determine icon and any other data needed to render the underlying `StatusIconAndText`.

Reply:
Done